custom_components/thermal_comfort/sensor.py

```diff
@@ -76,6 +79,16 @@
     "high",
 ]
 
+SIMMER_COOL= 'Cool'
+SIMMER_SLIGHTLY_COOL = 'Slightly cool'
+SIMMER_COMFORTABLE = 'Comfortable'
+SIMMER_SLIGHTLY_WARM = 'Slightly warm'
+SIMMER_INCREASING_DISCOMFORT = 'Increasing discomfort'
+SIMMER_EXTREMELY_WARM = 'Extremely warm'
+SIMMER_DANGER_OF_HEATSTROKE = 'Danger of heatstroke'
+SIMMER_EXTREME_DANGER_OF_HEATSTROKE = 'Extreme danger of heatstroke'
+SIMMER_CIRCULATORY_COLLAPSE_IMMINENT = 'Circulatory collapse imminent'
```

Comment:
Could you change those to short `snake_case` strings and add the proper string to sensor.en.json similar to the other translated strings?